### DISCUSSION TITLE
/settings Page Animation Preview

### DISCUSSION THREAD

**Discussion by sufyandaredevil** [monkeytypegame/monkeytype#4173] [Ideas] [answered]
I kind of noticed myself switching back n forth between the settings page and the type page right after I change a setting to check whether the style suits me, and I thought it would be better if there could a preview of what it might look like after I apply a setting with all the current configs I have setup. So it could be like when hovering over an option in the **/settings** menu using my **mouse** , I get a preview animation of what happens if I enable/disable that option.

An Example Scenario would be **Changing style of the caret**:
![Screenshot (21)](https://user-images.githubusercontent.com/45531461/231554788-c6719d45-8c60-4dbe-b83e-39cbc12f1f3c.png)

**Comment by unrealapex** [comment, marked as answer]
You could just change the setting in the command line and see its effect. Press <kbd>Esc</kbd> and then `Caret style > `

**Reply by sufyandaredevil** [reply]
Maybe a preview when pressing <kbd>Esc</kbd> and then the choosing a setting, also I've noticed something in the command line that there is no frequently applied setting listing in it just like we used to have in VSCode, I mean if I change the caret style the first time I need to retype it the next time instead of command line showing me in the listing at the top.
